src/main/java/net/william278/profanitycheckerapi/ProfanityChecker.java

```diff
@@ -91,4 +179,56 @@ public void dispose() throws IllegalStateException {
         close();
     }
 
+    private Set<String> getTextToCheck(String text) {
+        Set<String> toCheck = new HashSet<>();
+        toCheck.add(text);
+        toCheck.addAll(List.of(text.split(" ")));
+        Set<String> temp = new HashSet<>(toCheck);
+        for (String s : temp) {
```

Comment:
These variable namings get confusing to follow quickly, could we distinguish between where we're doing sets of strings and use `char`s where we're splitting the word.